tests/draft4/ref.json

```diff
@@ -174,6 +174,35 @@
         ]
     },
     {
+        "description": "nested ref with other properties",
+        "schema": {
+            "definitions": {
+                "a": {
+                    "type": "number"
+                }
+            },
+            "properties": {
+                "foo": {
+                    "type": "number",
+                    "maximum": 5,
+                    "$ref": "#/definitions/a"
+                }
```

Comment:
@epoberezkin The current behavior of `"$ref"` is clear and AFAIK not going to change in Draft 06.  Tests that test its current behavior should be accepted.